docs/_templates/custom-class-template.rst

```diff
@@ -0,0 +1,34 @@
+{{ fullname | escape | underline}}
+
+.. currentmodule:: {{ module }}
+
+.. autoclass:: {{ objname }}
+
+   {% block methods %}
+
+   {% if methods %}
+   .. rubric:: {{ _('Methods') }}
+
+   .. autosummary::
+      :toctree: _autosummary
+   {% for item in methods %}
+      {% if item != "__init__" %}
+      {{ name }}.{{ item }}
+      {% endif %}
+   {%- endfor %}
+   {% endif %}
+   {% endblock %}
+
+   {% block attributes %}
+   {% if attributes %}
+   .. rubric:: {{ _('Attributes') }}
+
+   .. autosummary::
+      :toctree: _autosummary
+   {% for item in attributes %}
+      {% if item.0 != item.upper().0 %}
```

Comment:
This checks if the first character is capitalized.  When we subclass a VTK class and autodocument that, we end up documenting all their attributes.  This is an easy of of avoiding that given how our methods and attributes are lowercase and theirs are capitalized.

`isupper` doesn't work in this case as it will only test if all characters are uppercase.

Reply:
Aaah, I see! Nice trick.